setup.cfg

```diff
@@ -65,9 +65,9 @@ tests =
 
 [flake8]
 ignore =
-    E501  # Line length handled by black.
```

Comment:
Should these comments perhaps be moved up instead of deleted? I find them quite useful. But not a blocker...

Reply:
I also wanted this, but one of the hooks deletes them 🤷